hacking/README.md

```diff
@@ -17,7 +17,7 @@ and do not wish to install them from your operating system package manager, you
 can install them from pip
 
     $ easy_install pip               # if pip is not already available
```

Comment:
Okay, on python-2.7.13 and python-3.5 (have to check older versions for pyhton2 but I'm pretty sure it wasn't in python-2.6.x) you can run this:
```
$ python -m ensurepip
```

to get pip and dependencies installed.